### PR TITLE
Feat/hakyll news pages migration

### PR DESCRIPTION
**Pull request by Maeevick**
__Topic__ : Migration for the [__news__ page](https://haskell.foundation/news/) from hardcoded html to metadata in markdown.

__Status :__ Open to Discussion / Ready for Review

__Next Steps :__
- _Generate News_
- _Generate Categories (date of publishing)_
- _Generate Grouped News by Category_
- _Need to format "date of publishing" (category)_

__Possible Improvements :__

- think about CSS layout _(grid, flex, nothing...?)_

### CODE REVIEW

File: templates/news/tile.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<div class="max-w-screen-xl mx-auto py-16 md:py-24">
+  <h2 class="text-2xl-4xl font-normal text-center">$category$</h2> <!-- I didn't formated the date for the moment to push fast I get feedbacks -->
+  $for(news)$
+  <!-- 
+    I had to make an arbitrary choice about grid layout as it was hardcoded : 
+    - I remove the 2 columns grid on large screen, to avoid complexe computation based on news' count
+    - Happy (or not) side-effect is removing the margin difference between november news and the others 
+  -->
```
Comment (Maeevick) [inline templates/news/tile.html:8]: need reflexion and discussion I think :thinking:

File: news/2020-11-05/haskell-foundation-announced.markdown [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Haskell Foundation Announced
+---
+
+Today Simon Peyton-Jones announced the formation of The Haskell Foundation, a non-profit organization focused on increasing adoption of the Haskell programming language. Watch the announcemnt <a href='https://youtu.be/MEmRarBL9kw' target='_blank'>here.</a>
```
Comment (myShoggoth) [inline news/2020-11-05/haskell-foundation-announced.markdown:5]: Does the embedded html thing work? I thought it was the [name](link) syntax?

Comment (Maeevick) [reply news/2020-11-05/haskell-foundation-announced.markdown:5]: It works on my side (and on other websites), if we want to keep the current behavior : "open in a new tab" with ```target='_blank'``` I don't know another solution than embed html in markdown

File: site.hs [hunk 3]
```diff
@@ -21,88 +21,149 @@ main = hakyllWith config $ do
...
 
+
+--------------------------------------------------------------------------------------------------------
+-- CONTEXT ---------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------------
+
+-- sponsors --------------------------------------------------------------------------------------------
+-- | Partition sponsors into by level: monad, applicative, and functor
+-- Sponsors are listed in the footer template, which means we need this
+-- context for most pages. The first argument is another context so
```
Comment (myShoggoth) [inline site.hs:105]: this comment will need to be updated

**Review by myShoggoth — CHANGES_REQUESTED** [review]
Just the comment update, everything else looks like a significant improvement.

File: site.hs (hunk 3 above)
Comment (Maeevick) [reply site.hs:105]: Oo sure, hawk eyes 👀 thanks for catching it

**Review by myShoggoth — APPROVED** [review]

**Pull request merged**